Makefile

```diff
@@ -179,6 +179,18 @@ code/lint:
 	@echo "--> Running golangci-lint"
 	@$(shell go env GOPATH)/bin/golangci-lint run --timeout 10m
 
+.PHONY: client/gen
+client/gen:
```

Comment:
We should document this in [README.md](https://github.com/keycloak/keycloak-operator/blob/2d19aae251da8f815fa00b370ea5c027cd994324/README.md#L118).